### DISCUSSION TITLE
Shortcut for "Show sidebar"

### DISCUSSION THREAD

**Discussion by ruko42** [strawberrymusicplayer/strawberry#1355] [Ideas]
A shortcut for switching the sidebar on/off would be helpful.

Yes, one can use the mouse to go to the menu, then Tools, then click "Show sidebar".  
For instance Ctrl + T is probably much faster. (... or Ctrl + B for toolBar) ...

( by the way TX for the really useful strawberry musicplayer! )

Rupert

**Comment by freugo54** [comment]
We definitely need that sidebar shortcut! As long as you don't want to change your playlists the sidebar information might be less important than the information in the various playlist columns, and with limited space on a small screen this can become a real issue. Rather than using Ctrl + B or + Ctrl T, I suggest to use F9 as this is most common to toggle side bars across other programs.
Also, the minimum width of the sidebar is much bigger than before in Clementine! Why is that? Again, on a small screen this is a problem as the visibility of playlist columns is unnecessarily reduced.

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
